src/archivist/services/pageDeclaration.js

```diff
@@ -38,4 +38,14 @@ export default class PageDeclaration {
 
     return [selector];
   }
+
+  toJSON() {
```

Comment:
I think `toJSON` is misnamed because it not only converts the representation to JSON but also renames its attributes to be the same as those found in service declarations.
What about toPersistenceFormat?